e2e/specs/patient-list.spec.ts

```diff
@@ -73,16 +73,24 @@ test('Manage patients in a list', async ({ api, page }) => {
     await patientListPage.goto(cohort.uuid);
   });
 
-  await test.step('Then I should be able to add and remove patients from that list', async () => {
+  await test.step('Then I should be able to add a patient to the list', async () => {
     // Add a patient to the list
     cohortMember = await addPatientToCohort(api, cohort.uuid, patient.uuid);
     await patientListPage.goto(cohort.uuid);
     await expect(patientListPage.patientListHeader()).toHaveText(/1 patients/);
     await expect(patientListPage.patientsTable()).toHaveText(new RegExp(patient.person.display));
+  });
 
-    // Remove a patient from the list
-    await removePatientFromCohort(api, cohortMember.uuid);
-    await patientListPage.goto(cohort.uuid);
+  await test.step('And then I should be able to remove a patient from the list', async () => {
+    await page.getByLabel(/remove from list/i).click();
+  });
+
+  await test.step("And then I click the 'Remove from list' to confirm", async () => {
+    await page.getByRole('button', { name: 'danger Remove from list' }).click();
```

Comment:
> danger Remove from list?

This looks weird 🤔

Reply:
That's how its done in elsewhere -https://github.com/openmrs/openmrs-esm-patient-management/blob/9201ed389bd8f811ff2c1f265c4ccbf60d5c3079/e2e/specs/appointments.spec.ts#L120 + what Playwright locator suggests
![Uploading Screenshot 2025-02-17 at 7.43.46 PM.png…]()